_posts/2025-02-10-announcing-ballerina-2201.11.0-swan-lake-update-11.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: blog-post
+title: Announcing Ballerina Swan Lake Update 11 (2201.11.0)
+author: Waruna Jayaweera
+published-date: 10 Feb 2025
+status: Published
+socialmediaimage: ballerina-generic-social-media-image-2023.png
+permalink: /posts/2025-02-10-announcing-ballerina-2201.11.0-swan-lake-update-11/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+The Ballerina team is excited to announce the eleventh update release of Ballerina Swan Lake, Ballerina Swan Lake Update 11 (2201.11.0). This update introduces enhancements across multiple areas, including Java 21 support, performance improvements, a new CSV module, an upgraded XML data module, and improvements to the Ballerina library and developer tools.
```

Comment:
since the CSV module and XML data module also belong to the `Ballerina library,` shouldn't we rephrase this?